github3/issues/event.py

```diff
@@ -58,6 +58,23 @@ def __init__(self, event, session=None):
         #: Dictionary of links for the pull request
         self.pull_request = event.get('pull_request', {})
 
+        #: Dictionary containing label details
+        self.label = event.get('label', {})
+
+        #: The integer ID of the event
+        self.id = event.get('id')
+
+        #: :class:`User <github3.users.User>` that is assigned
+        self.assignee = event.get('assignee')
+        if self.assignee:
+            self.assignee = User(self.assignee, self._session)
```

Comment:
You should be able to just do `User(self.assignee, self)` but it's not a big deal.